client/components/credit-card-form-fields/index.jsx

```diff
@@ -390,4 +381,28 @@ export class CreditCardFormFields extends React.Component {
 	}
 }
 
+function CardholderNameField( { createField, shouldRenderCountrySpecificFields, autoFocus } ) {
+	const translate = useTranslate();
+	const { isStripeLoading, stripeLoadingError } = useStripe();
+
+	const disabled =
+		isStripeLoading || stripeLoadingError ? ! shouldRenderCountrySpecificFields() : false;
```

Comment:
It would be helpful to add some parentheses here, since it's not obvious to me what the intended order of operations is.

Reply:
Good point. I just copy/pasted this logic from above, but it's definitely not clear. I pushed a change that extracts this logic to a function instead to make it explicit.